cpp/nvtx3.hpp

```diff
@@ -15,21 +15,67 @@
  */
 #pragma once
```

Comment:
Probably should get rid of this `pramga once` is it will likely interfere with all the custom include guard stuff.

Reply:
Yeah I was thinking about this, and I realized there is a case where it's a problem...   If someone includes nvtx3.hpp using the #define to disable the inlining, and then later someone else includes the exact same header without that define & uses the unversioned symbols, they'd be broken.  So yeah, I'll take out the #pragma once.

Reply:
done